domain/src/test/java/org/oppia/android/domain/topic/TopicListControllerTest.kt

```diff
@@ -285,14 +294,18 @@ class TopicListControllerTest {
       FRACTIONS_EXPLORATION_ID_1,
       getCurrentTimestamp()
     )
+    testCoroutineDispatchers.runCurrent()
 
-    val ongoingTopicList = retrieveOngoingStoryList()
-    assertThat(ongoingTopicList.recentStoryCount).isEqualTo(1)
-    verifyOngoingStoryAsFractionStory0Exploration1(ongoingTopicList.recentStoryList[0])
+    val promotedActivityList = retrievePromotedActivityList()
+    assertThat(promotedActivityList.promotedStoryList.recentlyPlayedStoryCount)
+      .isEqualTo(1)
+    verifyOngoingStoryAsFractionStory0Exploration1(
+      promotedActivityList.promotedStoryList.recentlyPlayedStoryList[0]
+    )
   }
 
   @Test
-  fun testRetrieveOngoingStoryList_markAllChaptersCompletedInFractions_ongoingStoryListIsCorrect() {
+  fun testGetPromotedStoryList_markAllChapsDoneInFractions_suggestedStoryListIsCorrect() {
```

Comment:
Please also add one test for the default case (e.g. no stories completed should suggest fractions + ratios).

Reply:
Added this testcase.

Reply:
I'm not seeing this test, did I miss it?

Reply:
This case is covered in `testGetStoryList_markRecentlyPlayedFirstTestTopic_defaultsuggestedStoryListIsCorrect`

Reply:
Ah, technically that's not exactly the same thing (since it's changing recently played), but I think it's reasonable.